packages/schema/src/config/common.ts

```diff
@@ -397,8 +442,10 @@ export default defineUntypedSchema({
   ignoreOptions: undefined,
 
   /**
-   * Any file in `pages/`, `layouts/`, `middleware/` or `store/` will be ignored during
-   * building if its filename starts with the prefix specified by `ignorePrefix`.
+   * Any file in `pages/`, `layouts/`, `middleware/`, `store/`, and `public/` directories will be ignored during
```

Comment:
I think `store` can be removed here (was Nuxt 2 specific). 
Besides that, it'd be good to check in the code which other parts are affected

Reply:
@manniL oh. I'm sorry. I ran lint:fix to see if stylistic changes are allowed.

Reply:
Oh, if they are from lint:fix that that's fine 🙈